slack-bot-message.el

```diff
@@ -36,12 +36,11 @@
                 bots)))
 
 (defmethod slack-bot-name ((m slack-bot-message) team)
-  (if (slot-boundp m 'bot-id)
-      (let ((bot (slack-find-bot (oref m bot-id) team)))
-        (if bot
-            (plist-get bot :name)
-          (oref m username)))
-    (oref m username)))
+  (or (oref m username)
```

Comment:
some bot messages (eg. from Github, Circleci) have blank string(eg. `""`) at username property.
I think it is better to use bot name if username is blank.
please add additional check whether username is blank or not. `slack-string-blankp` may be useful.

Reply:
done